apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -58,7 +60,7 @@ defmodule Aecore.Chain.Worker do
 
   @spec get_block_by_hex_hash(String.t()) :: Block.t()
   def get_block_by_hex_hash(hash) do
```

Comment:
We should also rename functions and variable names with "hex" in them if they are no longer hex. You can potentially remove some of such functions and variables. I would suggest using binary representation internally and bech for UI (including API and logs).